apps/heft/src/plugins/SassTypingsPlugin/SassTypingsGenerator.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import { render, Result } from 'node-sass';
+import postcss from 'postcss';
+import cssModules from 'postcss-modules';
+import { StringValuesTypingsGenerator, IStringValueTypings } from '@rushstack/typings-generator';
+import { LegacyAdapters } from '@rushstack/node-core-library';
+
+/**
+ * @public
+ */
+export interface ISassConfiguration {
+  /**
+   * Source code root directory.
+   * Defaults to "src/".
+   */
+  srcFolder?: string;
```

Comment:
I don't think it makes sense for this to be customizable.

Reply:
This one I kind of disagree with. The tsconfig.json file allows developers to make any folder the root of their source code. It could be `main` or `app/src`, etc. Therefore, Sass developers need the same flexibility.